mllib/src/main/scala/org/apache/spark/ml/tree/treeParams.scala

```diff
@@ -343,6 +343,8 @@ private[ml] trait RandomForestParams extends TreeEnsembleParams {
    *  - "onethird": use 1/3 of the features
    *  - "sqrt": use sqrt(number of features)
    *  - "log2": use log2(number of features)
+   *  - "(0.0-1.0]": use the specified fraction of features
+   *  - "[1-n]": use the specified number of features
```

Comment:
This note is a bit confusing. It isn't clear what n is and we don't mention that it should be smaller than `(number of features)`. I can't think of a great wording for this that is also concise. One option:
`"n": use n features, for integer 0 < n <= (number of features)`

I'm open to better wordings.